### ISSUE TITLE
Deploy key for cronjob is read-only

### ISSUE THREAD

**Issue by Beanow** [sourcecred/widgets#21]
On the nightly update of the widgets, pushing to the gh-pages branch fails because the `$SSH_DEPLOY_KEY` key is read-only.

https://circleci.com/gh/sourcecred/widgets/34

```
ERROR: The key you are authenticating with has been marked as read only.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Exited with code 128
```

**Comment by teamdandelion** [comment]
@Beanow I see you assigned me; can you give me a bit more context on how to resolve this, or what permissions you need in order to fix it directly?

**Comment by Beanow** [comment]
Yes, for the example you've configured a deploy key. I've written some about it on https://github.com/sfosc/sourcecred/blob/master/DEVELOPING.md#deploy-key we may reuse some parts of that for documenting here.

The key is used to push to the `gh-pages` branch, so it needs read-write permissions.
Github defaults to read-only when adding the public key, which is why the deploy fails on the cronjob.

To fix, I suggest generating a new keypair to replace the read-only one. As I previously found that removing and re-adding the same key with different settings causes all sorts of caching issues. Making sure to tick this box when adding the new public key to github.
![image](https://user-images.githubusercontent.com/497556/63587619-c3b15d00-c5a4-11e9-80f9-862cecf53ab3.png)

**Comment by Beanow** [comment]
The previous SSH key was configured in GitHub correctly to have write access but the line break issue resolved with #27 prevented it from being used.

After merging that I tried re-running the nightly, but this picked up the old config. _Fortunately_ I merged it just shy of 10 minutes before the nightly would trigger.

So here it is!
https://circleci.com/gh/sourcecred/widgets/90

I'll make a PR to remove the `--dry-run` so next time it will actually push to gh-pages.
